### ISSUE TITLE
Fine tune pod resources requirements

### ISSUE THREAD

**Issue by sampaccoud** [openfun/arnold#218]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**
When an OpenShift node reaches its resource limits, we end-up with 504 errors on some requests that a pod cannot handle.

**Describe the solution you'd like**
We need to help OpenShift determine how many pods of each type it can accept in order to make sure it will be able to run all requests.

See : https://docs.openshift.com/container-platform/3.10/dev_guide/compute_resources.html#dev-compute-resources

**Discovery, Documentation, Adoption, Migration Strategy**
Should the values be configurable via settings so we can adjust customer by customer depending on the usage that is made?

**Comment by jmaupetit** [comment]
Yes, I think making required resources configurable (but with relevant defaults) is required.
